mmv1/third_party/terraform/services/vertexai/resource_vertex_ai_feature_online_store_test.go

```diff
@@ -18,7 +18,6 @@ func TestAccVertexAIFeatureOnlineStore_updated(t *testing.T) {
 	acctest.VcrTest(t, resource.TestCase{
 		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
 		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
-		CheckDestroy:             testAccCheckVertexAIFeatureOnlineStoreDestroyProducer(t),
```

Comment:
Do you know why this is needed? I'm wondering if this means resources will accumulate in our test environment. If so, we probably need to alter the test.

Reply:
I suppose this function verifies if the resources are actually deleted after the terraform destroy is complete. But Our update test documentation mentions to remove this when creating the update test case from generated test files (https://googlecloudplatform.github.io/magic-modules/develop/test/#add-an-update-test). Hence, excluded.

Reply:
Ah, that could probably be a bit more clear. Those instructions are saying to delete the function definition, because it is already being autogenerated elsewhere in the package, but you want to keep the call here for the reason you mentioned (it makes sure the resource was deleted).

Reply:
Oh I see. Understood! Updated.